source/component/PasDoc_OptionParser.pas

```diff
@@ -321,14 +393,33 @@ destructor TOptionParser.Destroy;
 
 procedure TOptionParser.ParseOptions;
 var
-  LCopyList: TStringList;
-  i: Integer;
+  LCopyList, OptsFromFile: TStringList;
+  i, j: Integer;
   LFoundSomething: boolean;
 begin
   LCopyList := TStringList.Create;
   LCopyList.Assign(FParams);
   FLeftList.Clear;
+
   try
+    // Pre-process config files ("@<path-to-file>" parameters)
+    for i := LCopyList.Count - 1 downto 0 do
+      if LCopyList[i][1] = OptionFileChar then
+      begin
```

Comment:
Minor thing: Use `SCharIs(LCopyList[i], 1, OptionFileChar)` here instead of `LCopyList[i][1] = OptionFileChar`. This will not crash in case of an empty `LCopyList[i]` (which is possible if user calls like `pasdoc '' ...` in bash. Of course empty parameter is still invalid, but that should be reported elsewheere :)

Reply:
Nice note, done!